packages/docs/docs/theme/default-theme-config.md

```diff
@@ -72,6 +72,25 @@ module.exports = {
 }
 ```
 
+You can change `aria-label` of dropdown button to with a custom aria-label. Default value is text property. if the dropdown menu consists of language items., default value is 'Select language'
```

Comment:
I don't think we need an entire section for `aria-label`;) What I meant was just adding `ariaLabel: 'Language Menu'` in existing examples in this page. And I think you can do the same for Chineese

Reply:
Okey, i wil add that in existing examples :)